src/EditorFeatures/Core/Implementation/IntelliSense/SignatureHelp/Controller.Session_UpdateModel.cs

```diff
@@ -76,25 +80,25 @@ private static int GetSelectedParameter(SignatureHelpItem bestItem, int paramete
                     var result = filteredItems.FirstOrDefault(i => IsApplicable(i, argumentCount, name, isCaseSensitive));
                     if (result != null)
                     {
-                        return result;
+                        return (result, false);
                     }
 
                     // if we couldn't find a best item, and they provided a name, then try again without
                     // a name.
                     if (name != null)
                     {
-                        return GetBestItem(currentItem, filteredItems, selectedParameter, argumentCount, null, isCaseSensitive);
+                        return GetBestItem(currentItem, userSelected, filteredItems, selectedParameter, argumentCount, null, isCaseSensitive);
                     }
 
                     // If we don't have an item that can take that number of parameters, then just pick
                     // the last item.  Or stick with the current item if the last item isn't any better.
                     var lastItem = filteredItems.Last();
                     if (currentItem.IsVariadic || currentItem.Parameters.Length == lastItem.Parameters.Length)
                     {
-                        return currentItem;
+                        return (currentItem, false);
                     }
 
-                    return lastItem;
+                    return (lastItem, false);
```

Comment:
nit: i'm on the fence if we should use a named param for true/false in this method.  totally up to you.

Reply:
Changed to two ref parameters